selftests/unit/test_settings.py

```diff
@@ -41,6 +43,22 @@ def testFloatConversion(self):
     def testBoolConversion(self):
         self.assertTrue(self.settings.get_value('foo', 'bool_key', bool))
 
+    def testPathHomeDir(self):
+        raw_from_settings = '~/path/at/home'
+        path_from_settings = self.settings.get_value('foo', 'path', 'path')
+        self.assertEqual(path_from_settings[-13:],
+                         raw_from_settings[-13:])
+        self.assertGreaterEqual(len(path_from_settings),
+                                len(raw_from_settings))
+        home_from_environ = os.environ.get('HOME', None)
```

Comment:
I think it'd be better as it's default python way to provide the correct path. That way we risk less possible issue in the future (imagine windows, mac, solaris...)